### PR TITLE
adding `from` field in the sender email function

### PR DESCRIPTION
**Pull request by jyenes**
#### Background
notification from meca export are not being sent 

What does this PR do?
add `from` field on the configuration of the email

### CODE REVIEW

File: server/xpub-server/entities/manuscript/resolvers/submitManuscript.js [hunk 1]
```diff
@@ -35,24 +35,30 @@ async function submitManuscript(_, { data }, { user, ip }) {
...
       logger.info(`Manuscript ${manuscript.id} successfully exported`)
-      return Manuscript.updateStatus(
-        manuscript.id,
-        Manuscript.statuses.MECA_EXPORT_SUCCEEDED,
-      )
+      // return Manuscript.updateStatus(
+      //   manuscript.id,
+      //   Manuscript.statuses.MECA_EXPORT_SUCCEEDED,
+      // )
+      throw new Error('error, to test meca export')
```
Comment (hem-brahmbhatt) [inline server/xpub-server/entities/manuscript/resolvers/submitManuscript.js]: :shipit:

File: server/xpub-server/entities/manuscript/resolvers/submitManuscript.js [hunk 2]
```diff
@@ -35,24 +35,30 @@ async function submitManuscript(_, { data }, { user, ip }) {
...
     })
     .catch(async err => {
+      console.log('sending new error, with meca export.')
```
Comment (hem-brahmbhatt) [inline server/xpub-server/entities/manuscript/resolvers/submitManuscript.js]: You can never have enough logs.

**Review by hem-brahmbhatt — APPROVED** [review]

**Review by Cooryd — APPROVED** [review]

**Review by diversemix — APPROVED** [review]
Let's get this in ASAP

**Comment by diversemix** [comment]
@jyenes does this close the issue with emails not being sent? Or is there other associated work still to be done?

**Comment by hem-brahmbhatt** [comment]
> @jyenes does this close the issue with emails not being sent? Or is there other associated work still to be done?

I wasn't able to find any errors in the logs in productions associated with #1496, so I can't confirm if this will fix the issue. I suggest we monitor the situation and take a look tomorrow.

**Pull request merged**